<!DOCTYPE html>
<html>

<head>
    <title>Introductory Genetics Activities by Andrew Debevec</title>
    <link rel="stylesheet" type="text/css" href="normalize.css">
    <link rel="stylesheet" type="text/css" href="base.css">
</head>

<body>
    <div id="content">
        <h1>Introductory Genetics Activities</h1>
        <p class="center"><em>by <a href="https://www.andrewdebevec.com">Andrew Debevec</a></em></p>
        <p>The four activities before were created for the introductory biology course at the University of Illinois.
            The original versions of the first three activities, created by request of the instructor, were completed in
            2013 as part of a summer
            graduate assistantship. The most recent incarnations, linked below, were created in July 2023 as part of a
            modernization effort to ensure they would continue to be compatible with modern browsers as long as
            possible.</p>
        <p>While the original versions used the CreateJS package to interact with the canvas element, the new versions
            are more straightforward, using DOM elements to achieve the same purpose in a more accessible and compatible
            way.</p>
        <p>The fourth activity, featuring <em>Anopheles gambiae</em>, was created in 2015. While the code is not up to
            my current standard, it is still written in a way that allows it to be compatible and accessible, so it has
            not been updated.</p>
        <p>The images of <em>Drosophila melanogaster</em> and <em>Anopheles gambiae</em> were created by me, using
            public domain source images. The concept was cooperatively designed by Benjamin Clegg and myself, and the
            code was entirely written by myself, outside of the use of the <a
                href="https://necolas.github.io/normalize.css/" target="_blank"
                rel="noreferrer noopener">Normalize.CSS</a> library to ensure a consistent appearance across browsers
            and operating systems.</p>
        <ul>
            <li><a href="./monohybrid/index.html" target="_blank">Exploring Genotypic and Phenotypic Ratios in a
                    Monohybrid Cross</a><br />
                <span class="small_text">Go through several crosses of fruit flies, looking at alleles of the
                    <em>ebony</em> gene.</span>
            </li>
            <li><a href="./dihybrid/index.html" target="_blank">Exploring Genotypic and Phenotypic Ratios in a Dihybrid
                    Cross</a><br />
                <span class="small_text">Simulate crosses of flies with variations in the
                    <em>ebony</em> and <em>apterous</em>
                    genes, culminating in a full dihybrid cross.</span>
            </li>
            <li><a href="./linkagemaps/index.html" target="_blank">Linkage Maps in <em>Drosophila
                        melanogaster</em></em></a><br />
                <span class="small_text">This activity was designed for in-class use. Students are assigned a pair of
                    genes, and then select those two genes and simulate crosses of flies with those genes. Through this
                    activity, students test the null hypothesis that the genes independently assort, and will ultimately
                    reject that null hypothesis, as all four genes involved are linked. This naturally leads into a
                    discussion about linkage disequilibrium and crossing-over events during meiosis. Note: during the
                    class activity, passwords prevent students from continuing, in order to keep everyone at the same
                    place; these password checks have been disabled here, so simply click "Continue" to move
                    forward.</span>
            </li>
            <li><a href="./anopheles/index.html" target="_blank">Testing Linkage in <em>Anopheles
                        gambiae</em></em></a><br />
                <span class="small_text">While the other activities are mostly self-contained, this activity focuses on
                    the generation of data which is then used for a classwide activity when aggregated. Students should
                    be able to combine their data to make a full linkage map for the four genes featured in the
                    activity.</span>
            </li>
        </ul>
    </div>
</body>

</html>